test/unit/data/devices/list_devices_by_type.ts

```diff
@@ -0,0 +1,43 @@
+const request = {
+	url: 'https://api.smartthings.com/devices',
+	method: 'get',
+	headers: {
+		'Content-Type': 'application/json;charset=utf-8',
+		Accept: 'application/json',
+		Authorization: 'Bearer 52991afa-66e8-4af0-8d85-5c568ed5ba7d',
```

Comment:
We should make sure this is not a valid token.

Reply:
I'm assuming Bob did that already...I copied this from one of his examples (that's already committed).